packages/suite/src/middlewares/suite/analyticsMiddleware.ts

```diff
@@ -178,6 +180,25 @@ const analyticsMiddleware =
                     payload: { type: action.payload.walletNumber ? 'hidden' : 'standard' },
                 });
                 break;
+            case COINJOIN.SESSION_COMPLETED:
+            case COINJOIN.SESSION_PAUSE:
+            case COINJOIN.ACCOUNT_UNREGISTER: {
+                const anonymityGainToReport = selectAnonymityGainToReportByAccountKey(
+                    state,
+                    action.payload.accountKey,
+                );
+                if (anonymityGainToReport != null) {
```

Comment:
can it be `undefined`? 
Maybe i would check if it is type of number?

Reply:
a24f46da9017635a32bf89f3c6acb4c7e4c10d5d, changed to !==